### PR TITLE
chore(react): update snapshots and refactor code

### PR DESCRIPTION
**Pull request by savindi7**
### Purpose

Fix component name of `ButtonDropdownMenu` and aria controls in Button.
Update snapshots
Import and use wrapped components
Update stories
Fix review comments

### Related Issues
- N/A

### Related PRs
- https://github.com/wso2/oxygen-ui/pull/5

### Checklist
- [ ] UX/UI review done on the final implementation.
- [ ] Story provided. (Add screenshots)
- [x] Manual test round performed and verified.
- [ ] Unit tests provided. (Add links if there are any)
- [ ] Documentation provided. (Add links if there are any)

### Security checks
- [ ] Followed secure coding standards in http://wso2.com/technical-reports/wso2-secure-engineering-guidelines?
- [ ] Ran FindSecurityBugs plugin and verified report?
- [ ] Confirmed that this PR doesn't commit any keys, passwords, tokens, usernames, or other secrets?

### CODE REVIEW

File: packages/react/src/components/UserDropdownMenu/index.ts [hunk 1]
```diff
@@ -16,6 +16,7 @@
  * under the License.
  */
 
-export {default} from './ButtonDropdownMenu';
-export {ButtonDropdownMenuProps} from './ButtonDropdownMenu';
-export {ModeListInterface} from './ButtonDropdownMenu';
+export {default} from './UserDropdownMenu';
+export {UserDropdownMenuProps} from './UserDropdownMenu';
```
Comment (brionmario) [inline packages/react/src/components/UserDropdownMenu/index.ts:20]:
```suggestion
export type {UserDropdownMenuProps} from './UserDropdownMenu';
```

File: packages/react/src/components/UserDropdownMenu/index.ts [hunk 2]
```diff
@@ -16,6 +16,7 @@
...
+export {ModeListInterface} from './UserDropdownMenu';
```
Comment (brionmario) [inline packages/react/src/components/UserDropdownMenu/index.ts:21]: Lets ditch the Interface suffix.

File: packages/react/src/components/MenuItem/MenuItem.stories.mdx [hunk 3]
```diff
@@ -10,7 +11,13 @@ export const meta = {
...
 <Meta title={meta.title} component={meta.component} />
 
-export const Template = args => <MenuItem {...args} />;
+export const Template = (args) => {
+    return (
+        <Menu open={true} role="menu">
+            <MenuItem {...args} />
+        </Menu>
+    )
+};
```
Comment (thivi) [inline packages/react/src/components/MenuItem/MenuItem.stories.mdx:20]: I guess we are using 4 spaces here

Comment (savindi7) [reply packages/react/src/components/MenuItem/MenuItem.stories.mdx]: FIxed with https://github.com/wso2/oxygen-ui/pull/21/commits/59365a5711b8bf4b43fb857ef2b95c8446611b76

File: packages/react/src/components/UserDropdownMenu/index.ts (hunk 2 above)
Comment (savindi7) [reply packages/react/src/components/UserDropdownMenu/index.ts]: Fixed https://github.com/wso2/oxygen-ui/pull/21/commits/59365a5711b8bf4b43fb857ef2b95c8446611b76

File: packages/react/src/components/UserDropdownMenu/index.ts (hunk 1 above)
Comment (savindi7) [reply packages/react/src/components/UserDropdownMenu/index.ts]: Fixed https://github.com/wso2/oxygen-ui/pull/21/commits/59365a5711b8bf4b43fb857ef2b95c8446611b76

**Review by thivi — APPROVED** [review]

**Pull request merged**